packages/compass-aggregations/src/components/stage-editor/stage-editor.jsx

```diff
@@ -90,11 +89,6 @@ class StageEditor extends Component {
    * @param {String} value - The value of the stage.
    */
   onStageChange = (value) => {
-    if (this.props.stageOperator === null && value && value.charAt(0) === '[') {
-      this.props.newPipelineFromPaste(value);
-      this.props.runStage(0);
-      return;
-    }
```

Comment:
This was broken for awhile now (because stage operator is not `null` even when not selected), but recently we decided not to support this anymore https://jira.mongodb.org/browse/COMPASS-5855